src/.vuepress/theme/layouts/Layout.vue

```diff
@@ -77,7 +97,7 @@ export default {
       )
     },
 
-    shouldShowSidebar () {
+    shouldShowSidebar() {
```

Comment:
Apparently other functions are also following this convention. I simply forgot to turn off auto format when saving but I can revert these changes